sanity-io/structured-content-2022
# Adjust yellow colors

# Adjust yellow colors

## Intent

Adjust the orange/yellow colors according to the new design, as described in [Shortcut](https://app.shortcut.com/sanity-io/story/15376/updated-design-elements-consider-if-and-how-they-effect-the-brand)

## Description

This is basically just a global replace in the static SVG images, an update to the CSS variable definitions, and a simple color swap for the mobile menu as requested in the Shortcut story.

I didn't bother renaming the `-color-brand-yellow-light` variable to mirror the new Figma name (Brand/Yellow Light Beige), to limit the size of the PR diff.

I noticed that there is a contrast issue for the pricing table when links are included in the headers, but this is an issue before this PR as well (in fact, the new "Brand/Yellow" color seems to make it ever so slightly better). I'll follow this up separately.

## Testing this PR
1. Open the [front page](https://structured-content-2022-web-git-adjust-yellow-colors.sanity.build/)
2. Verify that the yellowy orange color of the conference logo, nav block and "CTA" buttons (with arrows in them) is consistent and slightly brighter than before, and matches the image in "Get conference updates" (this is the "orange" color)
3. Hover the Tickets link button and check that its background color becomes lighter (this is the "yellow" color)
4. Click and hold on the Tickets link and check that the background color becomes even lighter, matching the background of the "Get conference updates" section (this is the "yellow-light" color)

(The yellow can also currently be found in the column headers of the event pricing table on the [registration info](https://structured-content-2022-web-git-adjust-yellow-colors.sanity.build/registration-info) page, if you want to inspect in dev tools and compare the hex colors with Figma. Alternatively dev tools will typically have a function for forcing the `:hover` and `:active` states of an element, so that it's possible to move the mouse around and inspect these states.)
